multi/multi.go

```diff
@@ -321,18 +321,18 @@ func (connMulti *ConnectionMulti) Upsert(space interface{}, tuple, ops interface
 	return connMulti.getCurrentConnection().Upsert(space, tuple, ops)
 }
 
-// Call calls registered Tarantool function.
+// Call16 calls registered Tarantool function.
 // It uses request code for Tarantool 1.6, so result is converted to array of
 // arrays.
-func (connMulti *ConnectionMulti) Call(functionName string, args interface{}) (resp *tarantool.Response, err error) {
-	return connMulti.getCurrentConnection().Call(functionName, args)
+func (connMulti *ConnectionMulti) Call16(functionName string, args interface{}) (resp *tarantool.Response, err error) {
+	return connMulti.getCurrentConnection().Call16(functionName, args)
 }
 
-// Call17 calls registered Tarantool function.
+// Call calls registered Tarantool function.
 // It uses request code for Tarantool 1.7, so result is not converted
```

Comment:
Same here about specifying only 1.7 version in comment, We should re-check all the code about 1.7 references and rewrite them this way

Reply:
Done.